trace_decoder/Cargo.toml

```diff
@@ -54,38 +54,19 @@ glob = "0.3.1"
 libtest-mimic = "0.7.3"
 plonky2_maybe_rayon = { workspace = true }
 pretty_assertions = "1.4.0"
+zero = { workspace = true, features = ["eth_mainnet"] }
 pretty_env_logger = { workspace = true }
-prover = { workspace = true }
 serde_json = { workspace = true }
 serde_path_to_error = { workspace = true }
 
-
-[features]
-default = ["eth_mainnet"]
-eth_mainnet = [
-    "evm_arithmetization/eth_mainnet",
-    "prover/eth_mainnet",
-]
-cdk_erigon = [
-    "evm_arithmetization/cdk_erigon",
-    "prover/cdk_erigon",
-]
-polygon_pos = [
-    "evm_arithmetization/polygon_pos",
-    "prover/polygon_pos",
-]
-
 [[bench]]
 name = "block_processing"
 harness = false
-required-features = ["eth_mainnet"]
 
 [[test]]
 name = "consistent-with-header"
 harness = false
-required-features = ["eth_mainnet"]
 
 [[test]]
 name = "simulate-execution"
 harness = false
-required-features = ["eth_mainnet"]
```

Comment:
Why has this been removed? Running `simulate-execution` with a non-eth_mainnet feature would fail on the current payloads we have, and we already have feature-flagging in the decoder

Reply:
Thanks! I assumed that if our tests passed it wasn't needed

Reply:
> we already have feature-flagging in the decoder

I actually can't seem to find it on `develop`, not sure if it's been removed or not added yet but I have a WIP branch that does include the pre-state execution specific to `eth_mainnet` / `cdk_erigon` that requires feature-flagging in the decoder, so this should stay

Reply:
Yep, it seemed dead to me, adding this back